[2025-09-08T17:02:11.828+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-08T17:02:11.836+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskBranch scheduled__2025-09-07T00:00:00+00:00 [queued]>
[2025-09-08T17:02:11.840+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskBranch scheduled__2025-09-07T00:00:00+00:00 [queued]>
[2025-09-08T17:02:11.841+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-09-08T17:02:11.853+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): taskBranch> on 2025-09-07 00:00:00+00:00
[2025-09-08T17:02:11.856+0000] {standard_task_runner.py:72} INFO - Started process 16521 to run task
[2025-09-08T17:02:11.859+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill', 'taskBranch', 'scheduled__2025-09-07T00:00:00+00:00', '--job-id', '127', '--raw', '--subdir', 'DAGS_FOLDER/executing_cron_catchup_backfill.py', '--cfg-path', '/tmp/tmpib1c_mxc']
[2025-09-08T17:02:11.860+0000] {standard_task_runner.py:105} INFO - Job 127: Subtask taskBranch
[2025-09-08T17:02:11.888+0000] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskBranch scheduled__2025-09-07T00:00:00+00:00 [running]> on host codespaces-9bd7b3
[2025-09-08T17:02:11.935+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='loonycorn' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskBranch' AIRFLOW_CTX_EXECUTION_DATE='2025-09-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-07T00:00:00+00:00'
[2025-09-08T17:02:11.936+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-08T17:02:11.946+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-09-08T17:02:11.950+0000] {python.py:240} INFO - Done. Returned value was: taskC
[2025-09-08T17:02:11.950+0000] {branch.py:38} INFO - Branch into taskC
[2025-09-08T17:02:11.951+0000] {skipmixin.py:230} INFO - Following branch {'taskC'}
[2025-09-08T17:02:11.954+0000] {skipmixin.py:278} INFO - Skipping tasks [('taskE', -1)]
[2025-09-08T17:02:11.972+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-08T17:02:11.972+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskBranch, run_id=scheduled__2025-09-07T00:00:00+00:00, execution_date=20250907T000000, start_date=20250908T170211, end_date=20250908T170211
[2025-09-08T17:02:11.991+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-08T17:02:12.011+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-08T17:02:12.015+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-09-08T17:06:18.810+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-08T17:06:18.818+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskBranch scheduled__2025-09-07T00:00:00+00:00 [queued]>
[2025-09-08T17:06:18.822+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskBranch scheduled__2025-09-07T00:00:00+00:00 [queued]>
[2025-09-08T17:06:18.822+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-09-08T17:06:18.835+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): taskBranch> on 2025-09-07 00:00:00+00:00
[2025-09-08T17:06:18.838+0000] {standard_task_runner.py:72} INFO - Started process 18495 to run task
[2025-09-08T17:06:18.840+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill', 'taskBranch', 'scheduled__2025-09-07T00:00:00+00:00', '--job-id', '112', '--raw', '--subdir', 'DAGS_FOLDER/executing_cron_catchup_backfill.py', '--cfg-path', '/tmp/tmp8hl13zu5']
[2025-09-08T17:06:18.842+0000] {standard_task_runner.py:105} INFO - Job 112: Subtask taskBranch
[2025-09-08T17:06:18.870+0000] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskBranch scheduled__2025-09-07T00:00:00+00:00 [running]> on host codespaces-9bd7b3
[2025-09-08T17:06:18.912+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='loonycorn' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskBranch' AIRFLOW_CTX_EXECUTION_DATE='2025-09-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-07T00:00:00+00:00'
[2025-09-08T17:06:18.912+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-08T17:06:18.922+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-09-08T17:06:18.926+0000] {python.py:240} INFO - Done. Returned value was: taskC
[2025-09-08T17:06:18.927+0000] {branch.py:38} INFO - Branch into taskC
[2025-09-08T17:06:18.927+0000] {skipmixin.py:230} INFO - Following branch {'taskC'}
[2025-09-08T17:06:18.932+0000] {skipmixin.py:278} INFO - Skipping tasks [('taskE', -1)]
[2025-09-08T17:06:18.949+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-08T17:06:18.950+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskBranch, run_id=scheduled__2025-09-07T00:00:00+00:00, execution_date=20250907T000000, start_date=20250908T170618, end_date=20250908T170618
[2025-09-08T17:06:18.973+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-08T17:06:18.992+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-08T17:06:18.996+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
